examples/list.yaml

```diff
@@ -9,15 +9,21 @@ spec:
       elements:
       - cluster: engineering-dev
         url: https://1.2.3.4
+        values:
+          project: dev
       - cluster: engineering-prod
         url: https://2.4.6.8
+          values:
+            project: prod
       - cluster: finance-preprod
```

Comment:
L16-17: fix indentation

Reply:
Good catch, fixed. I also changed to use single quote instead of double quotes.